### DISCUSSION TITLE
Daily Challenge Medals

### DISCUSSION THREAD

**Discussion by rbmclean** [ppy/osu#29468] [Ideas]
There's quite a few decent medals that could be awarded for daily challenge participation.

- Play all maps in a week
- Place in the top X% Y times
- Play a total of X daily challenge maps
- Reach a daily/weekly streak of X
- ...

**Comment by Tenexxt** [comment]
I think that for max streak instead of a medal it could be a badge that would "level up" every time you reach a certain point, kinda like Mappers Guild badges.
